libmisc/user_busy.c

```diff
@@ -114,7 +140,6 @@ static int check_status (const char *name, const char *sname, uid_t uid)
 	FILE *sfile;
 
 	snprintf (status, 40, "/proc/%s/status", sname);
-	status[39] = '\0';
```

Comment:
Why did you get rid of this?

Reply:
Because snprintf always appends the NUL termination.

Reply:
Yes, you're right, and I guessthis step doesn't help with my real concern.  It's been bothering me that we don't check for if (ret < 0 || ret >= len).  All over the source.

But that's not your problem.  I think I'll change that in a separate PR.

Thanks.